homeassistant/components/conversation/default_agent.py

```diff
@@ -797,34 +783,32 @@ def _make_slot_lists(self) -> dict[str, SlotList]:
         _LOGGER.debug("Exposed entities: %s", entity_names)
 
         # Expose all areas.
-        #
-        # We pass in area id here with the expectation that no two areas will
-        # share the same name or alias.
         areas = ar.async_get(self.hass)
         area_names = []
         for area in areas.async_list_areas():
-            area_names.append((area.name, area.id))
-            if area.aliases:
-                for alias in area.aliases:
-                    if not alias.strip():
-                        continue
+            area_names.append((area.name, area.name))
+            if not area.aliases:
+                continue
+
+            for alias in area.aliases:
+                if not alias.strip():
```

Comment:
Since you expect excess spaces, shouldn't you also store the stripped version in `area_names` and `floor_names` ?

Reply:
Ah, good point :+1: